app/assets/stylesheets/stylesheets/lesson-content.css

```
@@ -102,6 +102,10 @@
     @apply text-current;
   }
 
+  .lesson-content > a > img:hover {
```

This doesn't appear to be working @Rllopez0410. I think this code will only apply to elements that are direct children of the .lesson-content class. But we want it to be applied to all matching content within the lesson content container.

Dropping the first `>` should fix it:

```suggestion
  .lesson-content a > img:hover {
```

---

ah, understood.